docs/src/developer/testing.md

```diff
@@ -31,47 +31,48 @@ test assertions. If you want to also measure test coverage, turn
 `PALACE_BUILD_WITH_COVERAGE` on. See [Unit test coverage](#Unit-test-coverage)
 for more details on this.
 
-Once the build completes, run the tests with:
+Once the build completes, the `palace-unit-tests` executable will be installed in the
+same `bin/` directory as the main `palace` executable. Run the tests with:
```

Comment:
Two thoughts:

1. We should explicitly call out the need to `make install` (or some equivalent) before running tests. I had plenty of people complain about tests not working and they were missing this step in a previous project.
2. Do we want to support running tests in the build directory w/o needing to install?